x/ecocredit/client/tx.go

```diff
@@ -23,18 +25,23 @@ func TxCmd() *cobra.Command {
 		Short: "Ecocredit module transactions",
 		RunE:  client.ValidateCmd,
 	}
-
 	cmd.AddCommand(
 		txCreateClass(),
+		txCreateBatch(),
+		txSend(),
+		txRetire(),
+		txSetPrecision(),
 	)
-
 	return cmd
 }
 
 func txCreateClass() *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "create_class [designer] [issuer[,issuer]*] [base64_metadata]",
-		Short: `Creates a new credit class.
+		Use:   "create_class [designer] [issuer[,issuer]*] [metadata]",
+		Short: "Creates a new credit class",
+		Long: `Creates a new credit class.
+
+Parameters:
   designer:  address of the account which designed the credit class
   issuer:    comma separated (no spaces) list of issuer account addresses. Example: "addr1,addr2"
   metadata:  base64 encoded metadata - arbitrary data attached to the credit class info`,
```

Comment:
In general I'd prefer if we had `--text-metadata` and `--binary-metadata` flags. JSON might be pretty common especially at this stage.

Reply:
But this can be in a separate PR

Reply:
OK, let's discuss it in a separate task: https://github.com/regen-network/regen-ledger/issues/179